vulture/core.py

```diff
@@ -55,6 +78,13 @@ def _get_unused_items(defined_items, used_names):
     return unused_items
 
 
+def _parse_error_codes(match):
+    # if no code is specified, assign it to `all`
```

Comment:
Start with upper case and end with full stop. I#d also be a little more concrete: 

If no error code is specified, add the line to the "all" category.